Formula/joplin.rb

```diff
@@ -1,8 +1,8 @@
 require "language/node"
 
 class Joplin < Formula
-  desc "Joplin, a free, open source note taking and to-do application"
-  homepage "http://joplin.cozic.net"
+  desc "A note taking and to-do application with synchronisation capabilities."
```

Comment:
The description cannot start with `A` or `An` and must not end with a period. :)